src/Service/SubcriptionService.php

```diff
@@ -80,7 +80,7 @@ public function subscripe($userData, Rooms $rooms, $isOrganizer, $group = array(
             if (!$isOrganizer) {
                 $this->notifier->sendNotification(
                     $this->twig->render('email/subscriptionToRoom.html.twig', array('room' => $rooms, 'subsription' => $subscriber)),
-                    $this->translator->trans('Bestätigung ihrer Anmeldung zur Konferenz: {name}', array('{name}' => $rooms->getName())),
+                    $this->translator->trans('Bestätigung ihrer Anmeldung zur Veranstaltung: {name}', array('{name}' => $rooms->getName())),
```

Comment:
Das bitte noch in der Translation finden und dort ändern

Reply:
Habs ergänzt